core/py/src/braintrust_core/span_types.py

```diff
@@ -0,0 +1,10 @@
+from enum import Enum
+
+
+class SpanTypeAttribute(str, Enum):
```

Comment:
Isn't there StrEnum to make this easier? https://docs.python.org/3/library/enum.html#enum.StrEnum